pkg/types/config.go

```diff
@@ -27,7 +27,7 @@ type SecretConfig struct {
 	Host string
 	Port int
 	// Path is the base path to the secret's location in the secret store
-	Path string
+	SecretName string
```

Comment:
This stays as `Path` since it is the actual Vault path to the service SecretStore. Maybe change to this:
```suggestion
	StoreBasePath string
```